ui-tests/src/test/java/org/openmrs/reference/CaptureVitalsTest.java

```diff
@@ -46,7 +46,8 @@ public void captureVitalsTest() {
         patientCaptureVitalsPage.setWeightField("78");
         patientCaptureVitalsPage.setTemperatureField("36.6");
         patientCaptureVitalsPage.setPulseField("120");
-        patientCaptureVitalsPage.setRespiratoryField("110");
+        patientCaptureVitalsPage.waitForPage();
+        patientCaptureVitalsPage.setRespiratoryField("99");
```

Comment:
Why the change in value from 110 -> 99?

Reply:
> Why the change in value from 110 -> 99?

Maximum value for respiratory rate should be 99.Thats according to the system limitation.

Reply:
@sherrif10 the maximum value of respiratory rate according according to our system is 999 not 99. See the test that validates the maximum value of the parameter [here](https://github.com/openmrs/openmrs-contrib-qaframework/blob/1d2bc4a9d4ccfdff98f49db1efd4f99d882cfabc/src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java#L122).

Reply:
> the maximum value of respiratory rate according according to our system is 999 not 99. See the test that validates the maximum value of the parameter [here](https://github.com/openmrs/openmrs-contrib-qaframework/blob/1d2bc4a9d4ccfdff98f49db1efd4f99d882cfabc/src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java#L122).

Thats what we are trying to resolve , please cross check again in the system

Reply:
For the past, the maximum value has been 999 for Respiratory rate! However, have just noticed a difference in what are servers are reading as per now.

https://demo.openmrs.org/ is reading 999 as was the value before
![VF](https://user-images.githubusercontent.com/38831673/133322906-4db28a38-4afc-44e2-a107-9e74ed18f58f.png)

https://qa-refapp.openmrs.org/openmrs/ is reading 99 which is a new value than what it was reading before
![UIx](https://user-images.githubusercontent.com/38831673/133323056-ebc6255f-9956-43ac-a12f-730c29146996.png)

Is this a change coming from the 2.12 release? Or Its a bug residing in our qa server making now two days old ?

Reply:
> Is this a change coming from the 2.12 release? Or Its a bug residing in our qa server making now one day old ?

Usually Demo sever contains old functionalities since its a released version, however a server is all time up to date with the current changes in referenceapplication-distro, when a change gets merged, immediately a server gets up to date FYI